packages/arb-validator-core/valprotocol/confirmation.go

```diff
@@ -21,68 +21,62 @@ import (
 	"math/big"
 )
 
-type ConfirmValidOpportunity struct {
+type ConfirmNodeOpp struct {
```

Comment:
I understand the desire to shorten the name, but "Opp" seems too ambiguous.  I could be "opposite" or "opposition".  Suggest Oppty, with a comment at the definition (or just stick with Opportunity.

Reply:
Updated